test-series/1/test-1.1.0.ipynb

```diff
@@ -0,0 +1,1263 @@
+{
```

Comment:
<p>"GLEAM diffuse sky model" -&gt; "GLEAM point source catalog".</p><p> </p><p>Should probably also say a little about how GLEAM was used, especially frequency structure. Was stokes-I interpolated from the subband fluxes? Was it treated as flat-spectrum using the total bandpass values? Were any cuts on the included sources made?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
These sound like questions for @zacharymartinot

Reply:
Fixed the typo about the point source catalog, but couldn't address the questions

Reply:
Some of these questions are answered here: http://reionization.org/wp-content/uploads/2020/04/HERA078_Calibration_vs__RIMEz.pdf

Reply:
<p>I've now filled in information about the GLEAM catalog in this paragraph using that linked file.</p>